Errors building buildmaster

When building buildmaster as described in the README I get

```
/home/zk261/nnpdf/buildmaster/filters/INT.cc: In function 'void register_integrability(std::vector<std::unique_ptr<Buildmaster::CommonData> >&)':
/home/zk261/nnpdf/buildmaster/filters/INT.cc:22:15: error: too many initializers for 'const std::array<std::__cxx11::basic_string<char>, 2>'
   22 |     "INTEGXV8"};
      |               ^
```

Others have gotten other errors in that file such as 

```
/buildmaster/filters/INT.cc:20:5: error: excess elements in struct initializer
    "INTEGXV",
    ^~~~~~~~~
1 error generated.
```

--- a/buildmaster/filters/INT.cc
+++ b/buildmaster/filters/INT.cc
@@ -14,7 +14,7 @@ void register_integrability(vector<unique_ptr<CommonData>>& list)
     "INTEGXV_543",
     "INTEGXV3_543",
     "INTEGXV8_543"};
-  const std::array<std::string, 2>  INTsets = {
+  const std::array<std::string, 5>  INTsets = {
     "INTEGXT3",
     "INTEGXT8",
     "INTEGXV",

